modules/network-observability-configuring-options-overview.adoc

```diff
@@ -7,10 +7,16 @@
 = Configuring advanced options for the Overview view
 You can customize the graphical view by using advanced options. To access the advanced options, click *Show advanced options*.You can configure the details in the graph by using the *Display options* drop-down menu. The options available are:
 
-* *Metric type*: The metrics to be shown in *Bytes* or *Packets*. The default value is *Bytes*.
 * *Scope*: To select the detail of components between which the network traffic flows. You can set the scope to *Node*, *Namespace*, *Owner*, or *Resource*. *Owner* is an aggregation of resources. *Resource* can be a pod, service, node, in case of host-network traffic, or an unknown IP address. The default value is *Namespace*.
 * *Truncate labels*: Select the required width of the label from the drop-down list. The default value is *M*.
 
 [id="network-observability-cao-managing-panels-overview_{context}"]
-== Managing panels
-You can select the required statistics to be displayed, and reorder them. To manage columns, click *Manage panels*.
\ No newline at end of file
+== Managing panels and display
+You can select the required panels to be displayed, reorder them, and focus on a specific panel. To add or remove panels, click *Manage panels*.
+
+The following panels are shown by default: 
+
+* *Top X average bytes rates*
+* *Top X bytes rates stacked with total*
```

Comment:
there are 2 more panels non-default panels which is not listed here. They are the packets rates one basically
Top X average packets rates*
*Top X packets rates stacked with total*

Reply:
These panels you mention are only available when packet drop feature is enabled, right? I added them to the packet drop section, I'll tag you there. Is it OK to put them in the Packet Drop Overview instead of the general Overview?

Reply:
no the above ones are non-default non-feature panels. 
There is no `drop` in the title listed above to be PacketDrop feature panels

Reply:
Ahh, ok thanks for explaining that :) I thought because they have packet in the name they are for packet drop.